compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensionsOps.td

```diff
@@ -475,4 +475,39 @@ def ApplyBufferOptimizationsOp :
   }];
 }
 
+def PackGreedilyOp : Op<Transform_Dialect,
+    "iree.pack_greedily",
+    [FunctionalStyleTransformOpTrait,
+     MemoryEffectsOpInterface,
+     TransformOpInterface,
+     TransformEachOpTrait]> {
+  let description = [{
+    Target the whole func op and rewrite known LinalgOp to packed form greedily.
+
+    Return modes:
+    =============
+    This operation ignores non-Func ops and drops them in the return.
+
+    It returns the same Func ops where all supported containing LinalgOps are
+    packed.
+  }];
+
+  let arguments = (ins PDL_Operation:$target);
+  let results = (outs PDL_Operation:$transformed);
```

Comment:
I'd recommend using `TransformHandleTypeInterface` or something similar and adding types to the syntax rather than hardcoding PDL operation.

Reply:
I never want to frontload fixing the MatchOp and I don't want to put casts everywhere in my tests .. so I'll bite the bullet and make the usptream change ..